Tasks/DockerComposeV0/dockercompose.ts

```diff
@@ -54,13 +56,16 @@ if (nopIfNoDockerComposeFile && !tl.exist(dockerFile)) {
                 "write service image digests": "./dockercomposedigests",
                 "combine configuration": "./dockercomposeconfig",
                 "run a docker compose command": "./dockercomposecommand"
-            }[action]).run(connection);
+            }[action]).run(connection, (pathToResult) => {
+                resultPaths += pathToResult;    
```

Comment:
You don't need this logic here. It can be a part of the utils class and avoid changing the constructs everywhere.

Reply:
Given that there's logic repeating everywhere. Adding this single line to the util should make it a little cleaner, what do you think?

Reply:
I think the logic should be here due to the fact that we don't limit one file to be written per command that's executed. Doing it like this allows the action action (for example dockercomposebuild) to add as much log files as it likes.